### DISCUSSION TITLE
How do I aggregate Paparazzi test reports across modules?

### DISCUSSION THREAD

**Discussion by erawhctim** [cashapp/paparazzi#1035] [Q&A]
If I have a bunch of UI modules, some that apply the Paparazzi plugin and some don't, is there a built-in way to aggregate all the paparazzi reports into one? Or would I have to apply the Paparazzi plugin to my root/app module to do so? 
(we're stuck behind on the Gradle 8.0 upgrade, so I can't upgrade Paparazzi to be able to apply it to our app module just yet)

I'm guessing I should write a custom Gradle task to try and do this?

**Comment by geoff-powell** [comment]
Currently there isn't a task to do this so a custom impl would be the suggested path. Will add it to issues as a potential enhancement.
